static/js/components/LearnerSearch.js

```diff
@@ -200,58 +200,63 @@ export default class LearnerSearch extends SearchkitComponent {
       <Card className="fullwidth" shadow={1}>
         <FilterVisibilityToggle
           {...this.props}
+          title="Course"
           filterName="courses"
         >
           <HierarchicalRefinementFilter
             field={"program.enrollments"}
-            title="Course"
+            title=""
```

Comment:
can we just omit the `title` prop from these filters, or do we have to pass in the empty string?